apps/email/index.html

```diff
@@ -32,6 +32,7 @@
   <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/tabs.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/value_selector.css" >
+  <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" >
```

Comment:
Actually, I didn't explain the whole story here (my bad). What motivated me in the first place was that arrows in settings buttons are broken since quite a long time: the image here does not exist any more: https://github.com/mozilla-b2g/gaia/pull/32697/files#diff-dcbffab9bed0fd786a4cd742c9c06e35L166 (there's several other occurrence of this).

In building blocks button, they have been replaced by gaia-icons if I'm not mistaken. So I think we have several choices:
1. don't bother here, leave it broken and fix in another bug (I can open one).
2. Fix it by replacing the images with a new one in `style/images/icons`, copied from gaia-icons and that will be mirrored in RTL.
3. Actually, the correct behaviour is the current one (no arrows displayed), so we just need to remove these dead codes.

I think I should do either 1 or 4, WDYT? 

I'm opening a bug for the gaia-icons story.